whylog/teacher/__init__.py

```diff
@@ -27,12 +83,16 @@ def update_pattern(self, line_id, proposed_pattern):
     def make_group(self, line_id, span):
```

Comment:
The idea was that user can add a group by choosing interval in text. It is the only purpose of this method.